app/models/order.py

```diff
@@ -17,6 +17,13 @@ def get_new_order_identifier():
         return get_new_order_identifier()
 
 
+def get_updatable_fields():
```

Comment:
Would it be a good practice to have this as a static attribute of the model itself?